site/content/docs/main/plugin-release-instructions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Releasing Velero plugins
+layout: docs
+toc: "true"
+---
+
+# Releasing Velero plugins
+
+Velero plugins maintained by the core maintainers do not have any shipped binaries, only container images, so there is no need to invoke a GoReleaser script.
+Container images are built via a CI job on git push.
+
+1. Update the README.md file to update the compatibility matrix and `velero install` instructions with the expected version number and open a PR.
+1. Once the updated README.md PR is merged, checkout the upstream `main` branch. `git checkout upstream/main`.
```

Comment:
The vSphere plugin does not use `main` as its default branch, it uses `master`. I also noticed that it doesn't have any GitHub releases, only tags. Are the core Velero team involved in that plugin's release process?

Reply:
We are not - that's a wholly separate team.

I should enumerate the plugins we're responsible as part of this PR.

Reply:
How about I link to the table at https://velero.io/docs/v1.5/supported-providers/ and mention that the vSphere plugin is an exception? Is that workable?

Reply:
Yeah, that works for me :) I had initially gone to that page but got a bit confused by the difference in the vSphere repo. I knew that there was a different team involved, but it wasn't clear if the core team was involved in the release process.

Reply:
We could also update https://velero.io/plugins/ to indicate that the vSpehere plugin is supported by vmware and not the velero team. That way this doc will not be relavent to the vsphere plugin.